### PR TITLE
Correct PR Route

### PR DESCRIPTION
**Pull request by g-boy05**

### CODE REVIEW

**Comment by deepu105** [comment]
same comment remove unwanted spaces

**Comment by JulienMrgrd** [comment]
(my #113 PR has changed a lot of code formatting)

**Comment by g-boy05** [comment]
I will see that

File: src/main/webapp/app/applications/applications.module.ts [hunk 1]
```diff
@@ -1,10 +1,10 @@
-import { NgModule, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
-import { RouterModule } from '@angular/router';
+import {NgModule, CUSTOM_ELEMENTS_SCHEMA} from '@angular/core';
```
Comment (deepu105) [inline src/main/webapp/app/applications/applications.module.ts]: formatting is wrong in all files. there should be space after { and before } here

**Comment by deepu105** [comment]
This is PR is snot required anymore as the corrections are done with root module

**Pull request closed without merging**